03/app.js

```diff
@@ -1,3 +1,19 @@
 export default function randomNumber(min, max) {
+  checkIfNumber(max);
+  checkScope(min, max);
+  const randomNum = Math.floor(Math.random() * (max - min)) + 1;
 
-}
\ No newline at end of file
+  return randomNum;
+}
+
+function checkIfNumber(max) {
+  if (typeof max !== 'number') {
+    throw new Error('Property have to be a number');
+  }
+}
```

Comment:
Nie wiem czy bym to zapisywał w oddzielnej funkcji, ponieważ chciałbym mieć osobny komunikat dla `min` oraz `max` :)